willow/image.py

```diff
@@ -266,6 +267,16 @@ def mime_type(self):
         return "image/heiс"
 
 
+class AvifImageFile(ImageFile):
```

Comment:
Having looked at a number of other prominent and widely used packages (e.g. Pillow), and the fact that the Python class naming convention recommend PascalCase, this is good.

The original classes have been like that since the very beginning and I would move towards renaming them, but that is for a separate PR